org.eclipse.dawnsci.analysis.api/src/org/eclipse/dawnsci/analysis/api/processing/OperationLog.java

```diff
@@ -0,0 +1,41 @@
+/*-
+ * Copyright 2017 Diamond Light Source Ltd.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.dawnsci.analysis.api.processing;
+
+/**
+ * Log of what happened during an operation
+ */
+public class OperationLog {
+	private static final String NEWLINE = "\n";
+	StringBuffer log = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)